### PR TITLE
EOS-26241: Set ADDB SIZE to 128M motr config file

### PR DESCRIPTION
**Pull request by atulsdeshmukh2312**
# Problem Statement
- EOS-26241: Set ADDB SIZE to 128M motr config file

# Design
-   log rotation failed, motr log file size went beyond 128MB
-

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py [hunk 1]
```diff
@@ -51,6 +51,7 @@
 MACHINE_ID_FILE = "/etc/machine-id"
 TEMP_FID_FILE= "/opt/seagate/cortx/motr/conf/service_fid.yaml"
 CMD_RETRY_COUNT = 5
+128M = 128 * 1024 *1024
```
Comment (yeshpal-jain-seagate) [inline scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:54]: s/*1024/* 1024/

File: scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py [hunk 2]
```diff
@@ -302,7 +303,8 @@ def update_copy_motr_config_file(self):
                    ("MOTR_M0D_DATA_DIR", f"{MOTR_M0D_DATA_DIR}"),
                    ("MOTR_M0D_CONF_XC", f"{MOTR_M0D_CONF_XC}"),
                    ("MOTR_M0D_ADDB_STOB_DIR", f"{MOTR_M0D_ADDB_STOB_DIR}"),
-                   ("MOTR_M0D_TRACE_DIR", f"{MOTR_M0D_TRACE_DIR}")]
+                   ("MOTR_M0D_TRACE_DIR", f"{MOTR_M0D_TRACE_DIR}"),
+                   ("MOTR_MOD_ADDB_RECORD_SIZE", f"{128M}")]
```
Comment (yeshpal-jain-seagate) [inline scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:307]: Why is this required as a hardcoded parameter, can it be taken from some conf parameters instead

Comment (atulsdeshmukh2312) [reply scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py:307]: If we dont want to harcode it then we can read it from solution.yaml as we are doing it with other config values.

**Comment by atulsdeshmukh2312** [comment]
Closing this PR.
New PR: https://github.com/Seagate/cortx-motr/pull/1295
